tests/console/umoci.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test umoci installation and basic usage
+#    Cover the following aspects of umoci:
+#      * Inspecting a repository before pulling it
+#      * Copy an image from Docker registry into an OCI image-spec
+#      * Unpack and repack (root/rootless) images
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use strict;
+use base "consoletest";
+use testapi;
+use caasp;
+
+sub run {
+    select_console("root-console");
+
+    record_info 'Test #1', 'Test: Installation';
+    my $package = 'umoci';
+    trup_install($package);
+
+    record_info 'Setup', 'Requirements';
+    assert_script_run('mkdir test_umoci && cd test_umoci');
+    assert_script_run('skopeo copy docker://docker.io/opensuse/tumbleweed oci:opensuse:tumbleweed', 300);
+
+    record_info 'Test #2', 'Test: Creating an image';
+    assert_script_run('umoci init --layout new_image');          # without tag
+    assert_script_run('umoci new --image new_image:new_tag');    # with tag
+
+    record_info 'Test #3', 'Test: Unpacking an image';
+    assert_script_run('umoci unpack --image opensuse:tumbleweed bundle');
+    assert_script_run("ls -l bundle | grep 'config.json\\|rootfs\\|sha256\\|umoci.json'");
```

Comment:
If you use egrep you don't have to escape pipes - but that's up to you, functionality is the same :)